web-test-runner.config.js

```diff
@@ -6,26 +6,17 @@ const defaultBrowsers = [
   playwrightLauncher({product: 'firefox', concurrency: 1}),
 ];
 
-const envBrowsers = process.env.BROWSERS?.split(',').map((product) =>
-  playwrightLauncher({product})
-);
+const envBrowsers =
```

Comment:
If we use Node 14 we don't need these changes

Reply:
Will test whether it just works, didn't want to get sucked into a big task of debugging random infra stuff.